#!/usr/bin/env python
PACKAGE = "arne_motion_simulator"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("linear_sensitivity", double_t, 0, "Increase for faster linear motion", 0.5, 0.0, 2)
gen.add("angular_sensitivity", double_t, 0, "Increase for faster rotation", 0.5, 0.0, 5)

exit(gen.generate(PACKAGE, "arne_motion_simulator", "MotionControl"))
